src/util/desktophelper.cpp

```diff
@@ -165,10 +165,23 @@ void DesktopHelper::openInFileBrowser(const QStringList& paths) {
         dirPath = dir.absolutePath();
         qDebug() << "opening:" << dirPath;
         if (!openedDirs.contains(dirPath)) {
-            QDesktopServices::openUrl(QUrl::fromLocalFile(dirPath));
+            openUrl(QUrl::fromLocalFile(dirPath));
             openedDirs.insert(dirPath);
         }
     }
 }
 
+bool DesktopHelper::openUrl(const QUrl& url) {
+#ifdef Q_OS_IOS
+    QUrl urlToOpen = url;
+    // Open files and folders in the iOS Files app
+    if (urlToOpen.scheme() == "file") {
+        urlToOpen.setScheme("shareddocuments");
+    }
```

Comment:
this could use a link to the official docs or smth.

Reply:
better than nothing I guess. though I'm very surprised that this is not official.